docs/src/markdown-pages/aptitudes/network.websocket.md

```diff
@@ -2,4 +2,33 @@
 name: "WebSocket Connect"
 links_js: "websocketconnect"
 ---
-Initiates a WebSocket connection. Returns a Socket object which can be used to write to the socket or set a message handler function.
\ No newline at end of file
+Initiates a WebSocket connection. Returns a Socket object which can be used to write to the socket or set a message handler function.
+
+To initailize a WebSocket connection, you'll first need to create a SocketConfiguration object. The `url` of the WebSocket is the only required field (see the [SocketConfiguration docs](https://open-olive.github.io/loop-development-kit/ldk/javascript/interfaces/socketconfiguration.html) for more configuration options). This URL must use the `wss://` scheme.
+
+```
+const socketConfig: SocketConfiguration = {
+    url: "wss://echo.websocket.org"
+}
+
+const socket: network.Socket = await network.webSocketConnect(socketConfig);
+```
+
+Once you have a Socket object, you can set a callback to handle incoming messages with `setMessageHandler`:
+```
+await socket.setMessageHandler(async (err, message) => {
+    // do something with message contents
+})
+```
+
+Write messages to the WebSocket with `writeMessage`:
+```
+await socket.writeMessage("Hello WebSocket!")
+```
+
+Set a callback for when the WebSocket is closed:
```

Comment:
Is the user responsible for closing their socket? I'm assuming there is a socket.close()?

Reply:
That is correct, you could do socket.close(). Although your socket could also be closed by the server and we want to give user a way to subscribe if this event will occur.

Reply:
@epiccoleman I think Michelle brought up a good point. Should we indicate that user could call socket.close() to kill the socket connection?